autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<blockquote>You may want to delete that once you're done working with it. </blockquote><p>I believe you're using SageMaker's default bucket, so to clarify, you could say that you may want to delete data within a certain prefix this example is using - probably don't want to delete the entire bucket.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>Ok.  Before I thought I was just deleting the stuff under the prefix.  That was based on something I cribbed from here: https://github.com/aws/amazon-sagemaker-examples/blob/main/autopilot/autopilot_california_housing.ipynb</p><p> </p><p>I've changed my comment to correctly state that you'd want to delete things under the prefix, not the whole bucket.</p><p> </p><p>Thanks</p>